dist/src/main/java/io/camunda/identity/IdentityModuleConfiguration.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.identity;
+
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.context.properties.ConfigurationPropertiesScan;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.FilterType;
+import org.springframework.context.annotation.Profile;
+
+/**
+ * Entry point for the Identity modules by using the the {@link
+ * io.camunda.application.Profile#IDENTITY} profile, so that the appropriate Identity application
+ * properties are applied.
+ */
+@ComponentScan(
+    basePackages = {"io.camunda.identity", "io.camunda.authentication"},
```

Comment:
When looking at the new architecture from a technical point of view, then `io.camunda.authentication` would belong to Camunda global/common. From ownership perspective it is fine having it here.